tests/regression_tests/ncrystal/test.py

```diff
@@ -69,6 +71,27 @@ def _get_results(self):
             df = tal.get_pandas_dataframe()
         return df.to_string()
 
+def test_cfg_from_xml():
+     """Make sure the cfg string is read by from_xml method"""
```

Comment:
An extra space here caused the CI test failure
```suggestion
    """Make sure the cfg string is read by from_xml method"""
```